src/listwiz/sorting.py

```diff
@@ -46,11 +46,21 @@ def merge_sort(l):
     return res
 
 
-def bubble_sort(l):
-    # We should provide bubble sort as well!
-    raise NotImplementedError
+def bubble_sort(arr):
```

Comment:
This would need some documentation of course.  At least for public facing functions.